### DISCUSSION TITLE
How to give a custom style to `subtitle:`?

### DISCUSSION THREAD

**Discussion by nipnipj** [quarto-dev/quarto-cli#2585] [Q&A]
We can customize `title:` header with:
```
h1.title {
  font-family: "Font name here";
}
```
How can we customize other header parts like `subtitle:` or `author`? Is there a public list of them?

**Comment by dragonstyle** [comment]
There is not a specific list. You have a couple of options - you can target them using css selectors (e.g. `p.subtitle {...}`) or you could provide your own title block template, which can emit any HTML that you'd like. More on customizing or replacing the title blocks here:

https://quarto.org/docs/authoring/title-blocks.html
